src/models/dashboard.model.ts

```diff
@@ -22,6 +22,10 @@ export interface Dashboard extends Document {
   archived: boolean;
   archivedAt?: Date;
   gridOptions?: any;
+  filterVariant?: string;
+  closable?: boolean;
+  contextualFilter?: any;
```

Comment:
no need to call that contextualFilter now if we have a filter property
we can name it like "structure"
and contextualFilterPosition can be just "position"

Reply:
We already have a property called "structure" in the dashboard model (used for widgets), I will call the contextualFilter of "filterStructure"